src/Components/FormCreation/CreateEditFormHeader.tsx

```diff
@@ -35,44 +38,162 @@ const DescriptionInput = styled(InputBase)`
   }
 `;
 
+const InputWrapper = styled.div`
+  padding: 48px 100px;
+  padding-left: 15%;
+  padding-right: 0px;
+`;
+
+const ButtonWrapper = styled.div`
+  padding: 48px 0px;
+  padding-left: 32px;
+  align-items: center;
+  justify-content: center;
+`;
+
+const DialogOverlay = styled.div`
+  position: fixed;
+  left: 0;
+  top: 0;
+  height: 150%;
+  width: 100vw;
+  z-index: 110;
+  background: rgba(0, 0, 0, 0.5);
+  .dialogButton {
+    text-transform: none;
+  }
+`;
+
 type HeaderData = {
   name: string;
   description: string;
+  previewLink: string;
 };
 
 type Props = {
   name: string;
   description: string;
+  previewLink: string;
   onChange: (data: HeaderData) => void;
 };
 
 function CreateEditFormHeader({
   name,
   description,
+  previewLink,
   onChange
 }: Props): React.ReactElement<typeof Header> {
   const onNameChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
I changed the logic slightly for this; I will handle the change when i rebase my pr